File: system-tests/debug-logging-test/src/test/java/com/teamscale/client/DebugLoggingTest.java
package com.teamscale.client;

import org.junit.jupiter.api.Test;

import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;

import static org.assertj.core.api.Assertions.assertThat;

public class DebugLoggingTest {

	private static final Path LOG_DIRECTORY = Paths.get("logTest").resolve("logs");

	@Test
	public void systemTest() throws Exception {
		assertThat(Files.exists(LOG_DIRECTORY)).isTrue();
		assertThat(Files.readAllLines(LOG_DIRECTORY.resolve("teamscale-jacoco-agent.log"))).isNotEmpty();
	}
}

File: system-tests/debug-logging-test/src/test/kotlin/com/teamscale/client/DebugLoggingTest.kt
package com.teamscale.client

import org.junit.jupiter.api.Test
import java.nio.file.Paths

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Assertions.assertTrue
import kotlin.io.path.exists
import kotlin.io.path.readLines

class DebugLoggingTest {
	@Test
	@Throws(Exception::class)
	fun systemTest() {
		assertTrue(LOG_DIRECTORY.exists())
		assertThat(LOG_DIRECTORY.resolve("teamscale-jacoco-agent.log").readLines()).isNotEmpty()
	}

	companion object {
		private val LOG_DIRECTORY = Paths.get("logTest").resolve("logs")
	}
}
